src/admin/python/lsst/qserv/admin/itest_load.py

```diff
@@ -73,6 +73,11 @@ def execute(cursor: MySQLCursor, stmt: str, multi: bool = False) -> None:
         cursors = [cursor]
 
     for cursor in cursors:
+        # Cast here because MySQLCursorAbtract does not have with_rows for some reason, even though both of
+        # its subclasses do...
+        cursor = cast(
+            Union[mysql.connector.cursor.MySQLCursor, mysql.connector.cursor_cext.CMySQLCursor], cursor
```

Comment:
Are you still at Python3.9? With 3.10 you can replace `Union` with `|` which is shorter and looks more natural.

Reply:
We're Python3.8, here, sadly!  (Qserv containers currently build off stock alma 8.)